v4/source/teams.yaml

```diff
@@ -968,3 +968,58 @@
                 type: string
         '400':
           $ref: '#/responses/BadRequest'
+
+'/teams/{team_id}/schemes/{scheme_id}':
+    put:
+      tags:
+        - teams
+      summary: Set a team's scheme
+      description: |
+        Set a team's scheme.
```

Comment:
Here I would be more explicit about what is really doing. Something like `Set the schema of the team with the id receive din the url as {team_id} to the schema with id received in the url as {schema_id}.`.

Without context is not so clear what is it doing.